.rvmrc

```diff
@@ -1 +1 @@
-rvm use 1.8.7-p352@cucumber-timed_formatter --create
+rvm use 2.2.3@cucumber-timed_formatter --create
```

Comment:
Did you stop using Ruby 1.8 in production? Why? :wink:

Also: Ruby 2.2.x is the minimum for current rails, right? I would like to keep this as low as possible.